### ISSUE TITLE
Multiple exports with the same name "load" for +layout.server.ts

### ISSUE THREAD

**Issue by austins** [HoudiniGraphql/houdini#792]
### Describe the bug

**Environment information:**
```
        "graphql": "^16.6.0",
        "houdini": "^0.19.2",
        "houdini-svelte": "^0.19.2",
        "@sveltejs/adapter-node": "^1.0.0",
        "@sveltejs/kit": "^1.0.1",
```

I have a `+layout.server.ts` without any Houdini GraphQL calls:

```ts
import type { LayoutServerLoad } from "./$types";

export const load = (() => ({ test: "Hello" })) satisfies LayoutServerLoad;
```

The following errors occur:
```
Multiple exports with the same name "load"
11 |  }
12 |  
13 |  export const load = (() => ({
   |               ^
14 |      test: "Hello"
15 |  })) satisfies LayoutServerLoad;

The symbol "load" has already been declared
11 |  }
12 |  
13 |  export const load = (() => ({
   |               ^
14 |      test: "Hello"
15 |  })) satisfies LayoutServerLoad;
```

I plan to use `+layout.server.ts` to access `locals` and compute other information for the app to pass to the layout. It seems that Houdini is breaking the load functions in this type of file.

### Severity

annoyance

### Steps to Reproduce the Bug

1. Create a `+layout.server.ts` with:
```ts
import type { LayoutServerLoad } from "./$types";

export const load = (() => ({ test: "Hello" })) satisfies LayoutServerLoad;
```

2. Refresh the site and see 500 Internal Error and the mentioned errors output in the server console.

**Comment by AlecAivazis** [comment]
Thanks for reporting this! I have a fix, will publish it shortly 👍

**Comment by AlecAivazis** [comment]
This should be fixed in `v0.19.3`

**Comment by austins** [comment]
That was fast! Thank you. Didn't know it detected the type of the function - thought it would go by the name of the function. Magical.

**Comment by AlecAivazis** [comment]
No problem at all. In general we process files by transforming their AST. it sounds a bit counter intuitive but its easier for me to do things that way than wrestle a bunch of regexes. Only down side is that we occasionally run into these kinds of edge cases. Thanks for being patient!

**Comment by austins** [comment]
Also, I can confirm this is fixed in `v0.19.3`. 😀

This is a small yet useful fix. It allows me to do refresh token auth logic in `hooks.server.ts` where cookies get updated, added, or removed, and provide the updated user object to `setSession()` and in an `event.locals.user` for layout use.

**Comment by sharmapukar217** [comment]
it seems that the issue still exists in houdini@v1.0.7
